webapp/pom.xml

```
@@ -65,31 +65,32 @@
 
     <dependencyManagement>
         <dependencies>
-            <dependency>
-                <groupId>life.qbic</groupId>
-                <artifactId>core</artifactId>
-                <version>0.2.0</version>
-            </dependency>
             <dependency>
                 <groupId>com.vaadin</groupId>
                 <artifactId>vaadin-bom</artifactId>
                 <version>${vaadin.version}</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
+                <groupId>life.qbic</groupId>
+                <artifactId>datamanager-core</artifactId>
```

since its the domain:
```suggestion
                <artifactId>datamanager-core</artifactId>
```
```suggestion
                <artifactId>datamanagent-core</artifactId>
```

---

typo @sven1103 ? You mean datamanagement.core?

---

Adapted and done